packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1543,13 +1543,16 @@ class SliverAppBar extends StatefulWidget {
       key: key,
       leading: leading,
       automaticallyImplyLeading: automaticallyImplyLeading,
-      actions: actions,
```

Comment:
how to implement `actions` if we provide `flexibleSpace` by using `FlexibleSpaceBar`?